lib/core/src/sdk.rs

```diff
@@ -1770,9 +1786,12 @@ impl LiquidSdk {
     }
 
     /// Receive from a Bitcoin transaction via a chain swap.
+    ///
+    /// If no `user_lockup_amount_sat` is specified, this is an amountless swap and `fees_sat` exclude
```

Comment:
```suggestion
    /// If no `payer_amount_sat ` is specified, this is an amountless swap and `fees_sat` exclude
```

Reply:
Essentially correct, but IMO the arg has to be adapted instead of the comment, as `user_lockup_amount_sat` is more accurate: d3f448819d81589e0a7b221880a419b54cebd9ca